tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -640,76 +644,100 @@ describe(__filename, () => {
       const root = render();
       expect(root.find('title')).toHaveLength(0);
     });
+  });
 
-    it('configures a rating manager', () => {
-      dispatchAddon(fakeAddon);
-      const location = createFakeLocation();
-      const root = render({ reviews: null, location });
+  describe('overallRatingStars', () => {
+    it('renders Rating without an add-on', () => {
+      const root = render({ addon: null });
+      const rating = root.find(Rating);
 
-      const manager = root.find(RatingManager);
-      expect(manager).toHaveLength(1);
+      expect(rating).toHaveProp('rating', null);
+    });
 
-      const instanceProps = root.instance().props;
-      expect(manager).toHaveProp('addon', instanceProps.addon);
-      expect(manager).toHaveProp('location', location);
-      expect(manager).toHaveProp(
-        'version',
-        instanceProps.addon.current_version,
-      );
-      expect(manager).toHaveProp('onReviewSubmitted');
+    it('renders Rating without add-on ratings', () => {
+      const addon = { ...fakeAddon, ratings: undefined };
+      dispatchAddon(addon);
+      const root = render();
+      const rating = root.find(Rating);
+
+      expect(rating).toHaveProp('rating', undefined);
     });
 
-    it('does not render a rating manager without a version', () => {
-      dispatchAddon({ ...fakeAddon, current_version: null });
+    it('renders Rating with add-on ratings', () => {
+      const addon = {
+        ...fakeAddon,
+        ratings: {
+          ...fakeAddon.ratings,
+          average: 4.5,
+        },
+      };
+      dispatchAddon(addon);
       const root = render();
+      const rating = root.find(Rating);
 
-      expect(root.find(RatingManager)).toHaveLength(0);
+      expect(rating).toHaveProp('rating', addon.ratings.average);
     });
 
-    it('handles a submitted review', () => {
-      const addonSlug = 'some-slug';
-      dispatchAddon({ ...fakeAddon, slug: addonSlug });
-      const dispatchSpy = sinon.spy(store, 'dispatch');
-      const root = render({ reviews: null, params: { addonSlug } });
+    it('renders RatingsByStar without an add-on', () => {
+      const root = render({ addon: null });
+      const ratingsByStar = root.find(RatingsByStar);
 
-      const manager = root.find(RatingManager);
-      expect(manager).toHaveProp('onReviewSubmitted');
-      const onReviewSubmitted = manager.prop('onReviewSubmitted');
+      expect(ratingsByStar).toHaveProp('addon', null);
+    });
 
-      // Simulate a review submission.
-      onReviewSubmitted();
+    it('renders RatingsByStar with an add-on', () => {
+      const addon = { ...fakeAddon };
+      dispatchAddon(addon);
+      const root = render();
+      const ratingsByStar = root.find(RatingsByStar);
 
-      sinon.assert.calledWith(dispatchSpy, clearAddonReviews({ addonSlug }));
+      expect(ratingsByStar).toHaveProp('addon');
+      expect(ratingsByStar.prop('addon')).toMatchObject({ id: addon.id });
```

Comment:
Why not just `expect(ratingsByStar).toHaveProp('addon', addon);`?

Reply:
Because the test doesn't access to the internal add-on object. I added a comment to make it less confusing.